### PR TITLE
Feature/add data node stuff

### PR DESCRIPTION
**Pull request by jeremyletang**
This revamp a little the folder + the generation of the protos.
- all proto sources are under sources
- all generated code end up at the top level under the same name than the sources.

### CODE REVIEW

File: buf.yaml [hunk 1]
```diff
@@ -3,7 +3,9 @@ version: v1beta1
 
 build:
   roots:
-    - vega
+    - sources
+    # - vega
+    # - data-node
```
Comment (karlem) [inline buf.yaml:8]:
```suggestion
    - vega
    - data-node
```

File: script/generate.sh [hunk 2]
```diff
@@ -24,11 +24,12 @@ function gen_code() {
 		&& ./script/fix_imports.sh "$pbfile"
 	done
 
-	chmod 0644 vega/*.go vega/api/*.go
+	#chmod 0644 vega/*.go vega/api/*.go
```
Comment (karlem) [inline script/generate.sh:27]: I guess this can just go?

Comment (jeremyletang) [reply script/generate.sh:27]: yes

**Review by karlem — APPROVED** [review]
Thanks for this Jeremy. It looks good to me.

**Pull request merged**